src/main/scala/com/databricks/labs/overwatch/ApiMeta.scala

```diff
@@ -177,3 +182,28 @@ class ClusterEventsApi extends ApiMeta {
   setApiCallType("POST")
   setStoreInTempLocation(true)
 }
+
+class JobRunsApi extends ApiMeta {
```

Comment:
This needs to use API v2.1 or we'll have issues. Please work with @Sriram-databricks to implement. You'll probably need to merge 0612_release branch into yours to pickup the changes you need to make this happen.

Reply:
added for job runs